cnd/src/config/file.rs

```diff
@@ -246,6 +255,15 @@ node_url = "http://localhost:8545/"
                     node_url: "http://localhost:8545".parse().unwrap(),
                 }),
             }),
+            lnd: Some(Lnd {
+                http_rpc_socket: Socket {
+                    address: IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)),
+                    port: 443,
+                },
+                macaroon: Some(PathBuf::from(
+                    "/home/jim/.lnd/data/chain/bitcoin/simnet/readonly.macaroon",
```

Comment:
Can we use a `~` instead?

Reply:
Done, thanks.

Reply:
Boy, blindly following questions burnt me on this PR.  `~/.lnd` is not valid on OSX for lnd data directory \me cries quietly   :)